docs/pages/docs/walkthroughs/lesson-1.md

```diff
@@ -87,7 +87,7 @@ export default config({
   db: {
     provider: 'sqlite',
     url: 'file:./keystone.db',
-  },
+  }
```

Comment:
```suggestion
  },
  lists: {} // ...
```

If the developer uses `keystone dev` without this, the error is:

```
TypeError: Cannot convert undefined or null to objec
```

Adding this changes the error into
```
Error:
You don't have any models defined in your schema.prisma, so nothing will be generated.
```

Which, in my opinion, is quite helpful when you are running keystone without Typescript